expressEngine/server.js

```diff
@@ -14,20 +14,22 @@ import Route from './route';
 // import docs
 import swaggerDocument from './doc/swagger.json';
 import seed from './db/seedData';
-import healthCheck from './lib/routeHelper';
+import routeHelper from './lib/routeHelper';
 
 // Set up the express app
 const app = express();
 config();
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
-app.use('/health-check', healthCheck);
+app.use(routeHelper.liveRoute(), routeHelper.liveRequest);
```

Comment:
@adilsuccessive Please move routes to route file

Reply:
Done with the changes